lit_llama/adapter_v2.py

```diff
@@ -26,20 +26,34 @@ def adapter_v2_state_from_state_dict(state_dict: dict) -> dict:
 
 
 def adapter_v2_new_forward(self, input: Tensor) -> Tensor:
+    weight = self.weight
+
+    try:
+        from lit_llama.quantization import Linear8bitLt
+        if isinstance(self, Linear8bitLt):
+            weight = self.dequantize(input.dtype)
+    except:
```

Comment:
What kind of exception are we expecting here? It would be useful to list them

Reply:
I don't know to be honest. I added this in case there are issues with the bitsandbytes library based on https://github.com/Lightning-AI/lit-llama/blob/main/lit_llama/quantization.py#L25

Reply:
Oh that's just so you can use the project without installing bitsandbytes

Reply:
Right, I think that should then be also true when using `adapter_v2.py`. I.e., without that modification, one would have to install bitsandbytes even if the person does not want to quantize.